configs/config_params.yaml

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+
+##USER NOTE: Certain fields are commented out as they are
+## not mandatory. Uncomment relevant fields based on your
+## use case.
+
+
+#Learning args
+clip_grad: 0 
+lr: 0.0001 
+epochs: 5
+afunc: relu 
+mse_weight: 0.001 
+pearson_weight: 1
+poisson_weight: 0
+# Experiment args
+label: 'AtacWorks'
+out_home: './Cache'
+train: False
+infer: False
+resume: False
+eval: False
+infer_files: ""
+intervals_file: ""
+sizes_file: ""
+infer_threshold:
+reg_rounding: 0
+cla_rounding: 3
+batches_per_worker: 16
+gen_bigwig: False
+weights_path:
+result_fname: 'infer_results.h5'
+deletebg: False
+# Training args
+task: 'both'
+train_files: ""
+print_freq: 10
+bs: 8
+num_workers: 4
+checkpoint_fname: ""
+save_freq: 1
+# Dataset args
+pad:
+transform: 'none'
```

Comment:
let's update this transform option to be 'None' as well so we have None standardized in our API (instead of one that's lower case only and one that's camel case)

or alternatively we can update the type_or_none_fn to compare against `str(val).lower()` so it's case insensitive

Reply:
done.

Reply:
One last request - looks like the "None" string isn't really checked in the transform logic (only the log option for now). Let's just take out 'None' from the choices for transform  (or add update 'none' to be 'None')

Reply:
So, in the case of transform.. it expects a string "none". Not the python NoneType object. My bad I didn't notice it before. It should be left as "none" so that it's not converted to NoneType by our function.

Reply:
done. removed "none" as a choice.